src/App.jsx

```diff
@@ -1,18 +1,73 @@
 import React from 'react';
-import NavBar from './components/NavBar/NavBar'
+import { createBrowserRouter, RouterProvider } from 'react-router-dom';
+import NavBar from './components/NavBar/NavBar';
 import Features from './components/Features/Features';
 import NewsLetter from './components/NewsLetter';
+import Footer from './components/Footer/Footer';
 import SearchBar from './components/SearchBar';
+import AboutUs from './components/AboutUs/AboutUs';
 
-function App() {
+const Home = () => {
   return (
     <div className="App">
       <NavBar />
       <SearchBar />
       <Features />
       <NewsLetter />
+      <Footer />
+    </div>
+  );
+};
+
+const AboutUsRoute = () => {
+  return (
+    <div>
+      <NavBar />
+      <AboutUs />
+      <Footer />
     </div>
   );
+};
+
+
+const NotFound = () => {
+  return (
+    <div>
+      <NavBar />
+
+      <div
+        style={{
+          display: 'flex',
+          justifyContent: 'center',
+          alignItems: 'center',
+          height: '100vh',
+          fontSize: '2rem',
+        }}
+      >
+        404 Not Found
+      </div>
+      <Footer />
+    </div>
+  )
+}
+function App() {
+  const router = createBrowserRouter([
+    {
+      path: '/',
+      element: <Home />,
+    },
+    {
+      path: '/about',
+      element: <AboutUsRoute />,
+    },
+    {
+      path: '*',
+      element: <NotFound />
+    },
+  ]);
+
+  return <RouterProvider router={router} />;
 }
 
 export default App;
+
```

Comment:
The app.js is not updated to the dev branch. plus, the about us page should be created in a different folder/file not here, then imported in app.js by routing and stuff. if you'd update ur branch it will be much easier since there is a folder for the pages and u'll get some insight.